CHANGELOG.md

```diff
@@ -1,5 +1,41 @@
 # Delta Chat Android Changelog
 
+## v1.0.0
+2019-12-17
+
+Finally, after months of coding and fixing bugs, here it is: Delta Chat 1.0 :)
+An overview over the changes since v0.510:
+
+* support for user avatars: select your profile image
+  at "My profile info" and it will be sent out to people you write to
+* introduce a new "Device Chat" that informs the user about app changes
+  and, in the future, problems on the device
+* new "Saved messages" chat
+* add "Certificate checks" options to "Login / Advanced"
+* if "Show classic emails" is set to "All",
+  emails pop up as contact requests directly in the chatlist
```

Comment:
afaict this is already built into 0.510.1 (at least)? My F-Droid installation has this feature already.

Reply:
the option was there, but the emails did not pop up as "contact requests" in the chatlist.